src/lib/components/LanguagesField.js

```diff
@@ -0,0 +1,48 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FieldLabel, SelectField } from 'react-invenio-forms';
+
+export class LanguagesField extends Component {
+  render() {
+    const { fieldPath, label, labelIcon, options, required } = this.props;
+    return (
+      <SelectField
+        fieldPath={fieldPath}
+        label={
+          <FieldLabel htmlFor={fieldPath} icon={labelIcon} label={label} />
+        }
+        options={options}
+        placeholder={'Select languages...'}
+        multiple
+        search
+        required={required}
+      />
+    );
```

Comment:
Let's use an array of SelectFields since we support multiple languages in the latest schema changes.

Reply:
The languages field uses the multiple prop from the dripdown component so you can select multiple languages... Do you think this isn't enough?

Reply:
Ah! My bad I didn't see that. That should work then.